dapps/admin/src/pages/listings/_EditScoring.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react'
+import { Button, Menu, MenuItem, Popover } from '@blueprintjs/core'
+
+import { query } from '../../utils/discovery'
+
+const ALL_TAGS = [
+  { tag: 'Super Featured', icon: 'clean' },
+  { tag: 'Featured', icon: 'star' },
+  { tag: 'High Quality', icon: 'thumbs-up' },
+  { tag: 'Normal', icon: 'circle' },
+  { tag: 'Low Quality', icon: 'thumbs-down' },
+  { tag: 'Hide', icon: 'ban-circle' },
+  { tag: 'Super Hide', icon: 'ban-circle' }
```

Comment:
What's the difference again between Hide and Super Hide ?
"Super Hide" is not super explicit :)... should it be renamed "Delete" ?

Reply:
I've renamed it to "Delete".